### ISSUE TITLE
Connecting dots 

### ISSUE THREAD

**Issue by stone16** [aleen42/How-elegant-a-common-point-is#1]
what does you imply saying common points? I seemingly did not see it in that issue. 

Do you mean that we could get innumerable knowledge and something creative from the reality of our daily life and inspire us to see the problem we face in different perspective?

If so, cola, viagra are all derivative in that way.

**Comment by aleen42** [comment]
yup, it's absolutely what you mean. Inspired from our daily life to solve different problems.

**Comment by aleen42** [comment]
yup, it's absolutely what you mean. Try to solve different problems with inspirations from our daily life.
